WebRoot/js/zimbraMail/calendar/view/ZmCalColView.js

```diff
@@ -437,7 +437,7 @@ function(hour) {
 
 ZmCalColView.prototype._updateTitle =
 function() {
-	var dayFormatter = DwtCalendar.getDayFormatter();
+	var dayFormatter = new AjxDateFormat(AjxMsg.formatDateMediumNoYear);
```

Comment:
Can we create a method in DwtCalendar to cache formatter object as was done previously

Reply:
Using AjxDateUtil._dateFormatNoYear; now.